tests/facts/windows_hotfixes/windows_hotfixes.json

```diff
@@ -0,0 +1,136 @@
+{
+    "command": "Get-CimInstance -ClassName Win32_QuickFixEngineering | Format-List",
+    "output": [
+        "",
+        "",
+        "Description         : Update",
+        "FixComments         : ",
+        "HotFixID            : KB4519569",
+        "InstallDate         : ",
+        "InstalledBy         : VAGRANT\\vagrant",
+        "InstalledOn         : 11/15/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "Description         : Update",
+        "FixComments         : ",
+        "HotFixID            : KB4462930",
+        "InstallDate         : ",
+        "InstalledBy         : VAGRANT\\vagrant",
+        "InstalledOn         : 11/15/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "Description         : Security Update",
+        "FixComments         : ",
+        "HotFixID            : KB4470788",
+        "InstallDate         : ",
+        "InstalledBy         : ",
+        "InstalledOn         : 3/12/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "Description         : Security Update",
+        "FixComments         : ",
+        "HotFixID            : KB4516115",
+        "InstallDate         : ",
+        "InstalledBy         : VAGRANT\\vagrant",
+        "InstalledOn         : 11/15/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "Description         : Security Update",
+        "FixComments         : ",
+        "HotFixID            : KB4523204",
+        "InstallDate         : ",
+        "InstalledBy         : NT AUTHORITY\\SYSTEM",
+        "InstalledOn         : 11/15/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "Description         : Security Update",
+        "FixComments         : ",
+        "HotFixID            : KB4523205",
+        "InstallDate         : ",
+        "InstalledBy         : NT AUTHORITY\\SYSTEM",
+        "InstalledOn         : 11/15/2019 12:00:00 AM",
+        "Name                : ",
+        "ServicePackInEffect : ",
+        "Status              : ",
+        "",
+        "",
+        "",
+        ""
+    ],
+    "fact": {
+        "windows_hotfixes": {
```

Comment:
I think the output here would be cleaner as:

```json
...
"fact": {
     "KB4519569": {
          "Description": "Update",
          "FixComments": "",
          "InstallDate": "",
          "InstalledBy": "VAGRANT\\vagrant",
          "InstalledOn": "11/15/2019 12:00:00 AM",
          "Name": "",
          "ServicePackInEffect": "",
          "Status": ""
     },
    ...
}
```

Reply:
Ok. Should those keys be sorted and lower-cased?

Like this:
...
"fact": {
     "KB4519569": {
          "description": "Update",
          "fixcomments": "",
          "installdate": "",
     },
    ...
}

Reply:
I think capitalized as it matches the Windows output. And I'd say unsorted, no real reason to sort them IMO.

Reply:
Ok